README.md

```diff
@@ -67,35 +78,46 @@ func main() {
     app := gofr.New()
 
     app.GET("/greet", func(ctx *gofr.Context) (interface{}, error) {
-
         return "Hello World!", nil
     })
 
-   app.Run() // listen and serve on localhost:8000 
+    app.Run() // listens and serves on localhost:8000 
 }
 ```
 
-To run the code, use the `go run` command, like:
+To run this code:
 
-```sh
+```bash
 $ go run main.go
 ```
 
-Then visit [`localhost:8000/greet`](http://localhost:8000/greet) in your browser to see the response!
+Visit [`localhost:8000/greet`](http://localhost:8000/greet) to see the result.
+
+---
+
+## 📂 **More Examples**
+
+Explore a variety of ready-to-run examples in the [GoFr examples directory](https://github.com/gofr-dev/gofr/tree/development/examples).
+
+---
+
+## 👩‍💻 **Documentation**
+
+- **[GoDoc](https://pkg.go.dev/gofr.dev)**: Official API documentation.
+- **[GoFr Documentation](https://gofr.dev/docs)**: Comprehensive guides and resources.
+
+---
 
-### See more examples
-A number of ready-to-run examples demonstrating various use cases of GoFr are available in the [GoFr examples](https://github.com/gofr-dev/gofr/tree/development/examples) directory.
+## 👍 **Contribute**
 
-## 👩‍💻Documentation
-See the  [godocs](https://pkg.go.dev/gofr.dev).
+Help us make GoFr even better:
 
-The documentation is also available on  [gofr.dev](https://gofr.dev/docs).
+1. **Star** this repo on GitHub! 🌟
+2. Write a review or tutorial on **[Medium](https://medium.com/)**, **[Dev.to](https://dev.to/)**, or your blog.
+3. Review the **[CONTRIBUTING.md](CONTRIBUTING.md)** guide to learn how to contribute to the project.
 
-## 👍 Contribute
-If you want to say thank you and/or support the active development of GoFr:
+---
 
-1. [Star](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars) the repo.
-2. Write a review or tutorial on [Medium](https://medium.com/), [Dev.to](https://dev.to/) or personal blog.
-3. Visit [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
+### 🎁 **Get a GoFr T-Shirt & Stickers!**
 
-If your PR is merged or you have written an article or contributed in someway to development or spreading the word about GoFr, fill the [Google Form](https://forms.gle/R1Yz7ZzY3U5WWTgy5), and we will send you a <b>GoFr T-Shirt and Stickers</b> as a token of appreciation.
+If your PR gets merged or you contribute through articles or spreading the word about GoFr, fill out [this form](https://forms.gle/R1Yz7ZzY3U5WWTgy5) to claim your **GoFr T-shirt** and stickers as a thank-you gift! 
```

Comment:
I feel you can write it like this,

If your PR is merged, or if you contribute by writing articles or promoting GoFr, we invite you to fill out [this form](https://forms.gle/R1Yz7ZzY3U5WWTgy5) to claim your **GoFr merchandise** as a token of our appreciation!

Reply:
Sure.

Reply:
@coolwednesday The update provided by you is resolved.